src/snippets/filters/reference/sveltekit/Allow.ts

```diff
@@ -0,0 +1,33 @@
+import { env } from "$env/dynamic/private";
+import arcjet, { filter } from "@arcjet/sveltekit";
+import { type RequestEvent, error } from "@sveltejs/kit";
+
+const aj = arcjet({
+  key: env.ARCJET_KEY!, // Get your site key from https://app.arcjet.com
+  rules: [
+    filter({
+      allow: [
+        // Requests matching this expression will be allowed. All other
+        // requests will be denied.
+        'http.request.method eq "GET" and ip.src.country eq "US" and not ip.src.vpn',
+      ],
+      mode: "LIVE",
+    }),
+  ],
+});
+
+export async function handle({
+  event,
+  resolve,
+}: {
+  event: RequestEvent;
+  resolve: (event: RequestEvent) => Response | Promise<Response>;
+}): Promise<Response> {
```

Comment:
In practice I think this would look different as sveltekit generates type files for route handlers like this to use

Reply:
I copied this from other SvelteKit examples - do you have an example already in our docs where this is more correct? Maybe needs wider fixes across the docs?